dex/src/main/scala/com/wavesplatform/dex/AddressActor.scala

```diff
@@ -551,6 +553,14 @@ object AddressActor {
 
   private case class InsufficientBalanceOrder(order: Order, insufficientAmount: Long, assetId: Asset)
 
+  sealed abstract class OrderListType(val hasActive: Boolean, val hasClosed: Boolean) extends Product with Serializable
+  object OrderListType {
+    case object All        extends OrderListType(true, true)
+    case object Empty      extends OrderListType(false, false)
```

Comment:
![image](https://user-images.githubusercontent.com/40212703/80375136-74909a00-88a0-11ea-95a0-eae2eb01c39a.png)

Reply:
Yeah, it is possible :)